aten/src/THC/THCBlas.cu

```diff
@@ -271,14 +271,24 @@ void THCudaBlas_HgemmStridedBatched(THCState *state, char transa, char transb, i
                                    (int) batchCount, rocblas_datatype_f32_r, rocblas_gemm_algo_standard,
                                    0, 0));
 #else
+#if defined(CUDA_VERSION) && CUDA_VERSION < 11000
```

Comment:
Are these checks still needed here?

Reply:
Yes.

Reply:
Yes, I think so.  If we're compiling with Cublas < 11, we still need to set CUBLAS_TENSOR_OP_MATH explicitly to enable Tensor Cores.